README.md

```diff
@@ -24,10 +29,51 @@ require('@newrelic/mysql')
 - [`mysql`](https://www.npmjs.com/package/mysql)
 - [`mysql2`](https://www.npmjs.com/package/mysql2)
 
-For more information, please see the agent [installation guide][3], and
-[compatibility and requirements][4].
+For more information, please see the agent [installation guide][3] and the agent [compatibility and
+requirements][4].
+
+Our [API and developer documentation](http://newrelic.github.io/node-newrelic/docs/) for writing
+instrumentation will be of help. We particularly recommend the tutorials and various "shim" API
+documentation.
+
+## Testing
+
+The module includes a suite of functional tests which should be used to verify that your changes
+don't break existing functionality.
+
+These tests rely on a Docker container being setup via scripts in the [Node.js
+agent](https://github.com/newrelic/node-newrelic). This container can be setup by running `npm run
+services` in the main agent repo.
+
+All tests are stored in `tests/` and are written using [Tap](https://www.npmjs.com/package/tap) with
+the extension `.tap.js`.
+
+To run the full suite, run: `npm test`.
+
+## Support
+
+New Relic hosts and moderates an online forum where you can interact with New Relic employees as
+well as other customers to get help and share best practices. Like all official New Relic open
+source projects, there's a related Community topic in the New Relic Explorers Hub. You can find this
+project's topic/threads here: https://discuss.newrelic.com/c/support-products-agents/node-js-agent/.
+
+## Contributing
```

Comment:
Did we leave out the privacy and vulnerability stuff on purpose because this is experimental? Feels to me like we'd still want those? Unless we were directed not to.

I'd have also been fine with leaving in the roadmap but given experimental... seems fine to not have that.

Reply:
I'm not sure why privacy and vulnerability stuff was left off.  I can add it back in with follow up PR into main.